### ISSUE TITLE
CI assigns a unique URL with build info to every build (P7)

### ISSUE THREAD

**Issue by AmandaStromdahl** [PersonligaPersson/DD2480_Group_27_CI#11]
Every time the CI builds the program, the build information should be saved with a unique URL. This information needs persist even if the CI server were to be restarted.

**Comment by PersonligaPersson** [comment]
Pull request #27 creates .txt files wih log information from the builds. We should be able to expose the logfiles, thus serving as a unique url for each build.

**Comment by PersonligaPersson** [comment]
This issue can be considered closed now after #27 and #12 as the logging functionality has been merged into the main branch.
